src/pip/_internal/commands/check.py

```diff
@@ -22,20 +23,16 @@ def run(self, options, args):
         for project_name in missing:
             version = package_set[project_name].version
             for dependency in missing[project_name]:
-                logger.info(
-                    "%s %s requires %s, which is not installed.",
-                    project_name, version, dependency[0],
```

Comment:
I think you should retain the existing indentation here. That way only the first line (the one with `logger.info(`) will show a difference.

Reply:
I am happy to revert this back but its kinda odd since the rest of the string first on the line.     

it seems a little inconsistent some files always break if its over a single line others never break unless its more then the allowed char length.